README.md

```diff
@@ -130,4 +130,8 @@ Join the [Discord server](discord.gg/melkeydevhouse)! We're always around to hel
 
 You can also probably catch Melkey working live on [Twitch](twitch.tv/melkey)! 
 
-Pop in and say hi! ♡
\ No newline at end of file
+Pop in and say hi! ♡
+
+## License
+
+Licensed under [MIT License](https://github.com/Melkeydev/rag-stack-lambda/LICENSE)
```

Comment:
This is not the best way to link to the LICENSE.
./LICENSE
is all you need relative from the ReadMe.md